projekt/src/app/task/task.component.spec.ts

```diff
@@ -1,24 +1,42 @@
+import { APP_BASE_HREF } from '@angular/common';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { AppModule } from '../app.module';
+import { Project } from '../models/project';
+import { Task } from '../models/task';
 
 import { TaskComponent } from './task.component';
 
 describe('TaskComponent', () => {
   let component: TaskComponent;
-  let fixture: ComponentFixture<TaskComponent>;
 
-  beforeEach(async () => {
-    await TestBed.configureTestingModule({
-      declarations: [TaskComponent],
-    }).compileComponents();
-  });
+  // testdata for the tests
+  const testProject = new Project('test Project');
+  testProject.tasks = [new Task('test task 1'), new Task('test task 2')];
+  testProject.tasks[0].status = 1;
 
   beforeEach(() => {
-    fixture = TestBed.createComponent(TaskComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
+    TestBed.configureTestingModule({
+      imports: [AppModule],
+      providers: [{ provide: APP_BASE_HREF, useValue: '/' }, TaskComponent],
+    });
+    component = TestBed.inject(TaskComponent);
   });
 
   it('should create', () => {
```

Comment:
missing test description

Reply:
thees are auto generated tests. I deleted them in each file now because they are not needed.